src/test/resources/templates/forms.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:hx="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Forms Test</title>
+
+    <style>
+        .htmx-indicator {
+            opacity: 0;
+        }
+
+        .htmx-indicator.htmx-request {
+            opacity: 1;
+            animation: ease;
+            animation-duration: 1s;
+        }
+
+        .error {
+            color: red;
+        }
+    </style>
+</head>
+<body>
+<h3>Signup Form</h3>
+<!--/*@thymesVar id="body" type="io.github.wimdeblauwe.hsbt.mvc.FormsController.FormValues"*/-->
+<form hx-post="/forms/newAccount" th:fragment="formBody" th:object="${body}">
+    <div hx-target="this" hx-swap="outerHTML" th:fragment="emailField" th:object="${body}">
+        <label>Email Address</label>
+        <input name="email" hx-post="/forms/newAccount" hx-indicator="#ind" th:field="*{email}">
```

Comment:
For come reason the `*{email}` works when using the fragment directly, but fails when using the `multi-fragment` template and using a `th:include` for the fragment. Any idea why that might happen? Does the second iclude get a separate context?

Reply:
Good question. I don't know. Maybe you can ask in the Thymeleaf GitHub project?

Reply:
I got it working. Likely was breaking because I was using the deprecated `th:include`, Now with `th:insert` I'm good to go.